zng/docs/spec.md

```diff
@@ -100,47 +100,55 @@ The ZNG type system comprises the standard set of primitive types like integers,
 floating point, strings, byte arrays, etc. as well as container types
 like records, arrays, and sets arranged from the primitive types.
 
-For example, a ZNG stream representing the single string "hello world"
-looks like this:
+For example, a TZNG stream representing the single string "hello world"
+may look like this:
 ```
-9:hello, world
+#35:string
+35:hello, world
 ```
-Here, the type ID is the integer "9" representing the string type
-(defined in [Typedefs](#typedefs)) and the data value "hello, world"
-is an instance of the string type.
+Here, the first line binds a tag `35` to the ZNG `string` data type
+and the second line references that tag to specify a value of the `string`
+type. TZNG is being shown here for human readability. In the equivalent
+binary ZNG encoding, a single [Value Message](#32-value-messages) can represent
+the string by referencing the type ID of `9` (per the
+[Primitive Types](#5-primitive-types) table) with no leading
+[Typedef](#311-typedefs) required.
 
-ZNG gets more interesting when different data types are interleaved in the stream.
-For example,
+ZNG gets more interesting when different data types are interleaved in the
+stream.  For example, in the following TZNG:
 ```
-9:hello, world
-4:42
-9:there's a fly in my soup!
-9:no, there isn't.
-4:3
+#35:string
+35:hello, world
+#36:int64
+36:42
+35:there's a fly in my soup!
+35:no, there isn't.
+36:3
 ```
-where type ID 4 represents an integer.  This encoding represents the sequence of
-values:
+Here the tag `36` now binds to one of ZNG's integer types. This encoding
+represents the sequence of values that could be expressed in JSON as:
```

Comment:
```suggestion
represents the sequence of values that could be expressed in JSON as
```
A colon doesn't make sense to me here.

Reply:
Hmm, ok. I flipped through the rest of the spec and in _most_ places we use some kind of punctuation before a code block that shows example text, but I see your point in that sometimes the sentence fragments seem to justify it and sometimes they maybe don't. I spotted one other place where we went straight from an "as" to a block without punctuation, so I'm happy with your suggestion. FWIW there's another place in the Section 4 "Examples" where we do nearly the same lead-in to a bit of comparative JSON, so I'll make the same change there.